### ISSUE TITLE
Lockout in React app template with Identity doesn't seem to be working.

### ISSUE THREAD

**Issue by azydevelopment** [aspnet/AspNetCore#17575]
### Describe the bug
Lockout won't work when using the React app template with Identity (Visual Studio 2019). The Login page when scaffolded in does show this:

![image](https://user-images.githubusercontent.com/29720335/70125364-89d40600-162b-11ea-8515-423bad460cd3.png)

Two things to note:

1. The lockoutOnFailure argument is set correctly
2. The documentation in the comments just above that call seem to be wrong

### To Reproduce
Use this template: 

![image](https://user-images.githubusercontent.com/29720335/70125577-ec2d0680-162b-11ea-9e2c-6ebbd8a7f85e.png)

Register a new user account.
Try to log in multiple times.

Expected behaviour: After the [default number of failed login attempts](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity-configuration?view=aspnetcore-3.1) the user should be locked out.
Actual behaviour: The user can continuously try to log in.

**Comment by HaoK** [comment]
So the scaffolding was out of sync from the UI code, which doesn't set lockout to true: see https://github.com/aspnet/AspNetCore/blob/master/src/Identity/UI/src/Areas/Identity/Pages/V4/Account/Login.cshtml.cs#L132

Did you confirm that the lockout works after scaffolding?

**Comment by azydevelopment** [comment]
Hmm. It seems to work now in my custom project. So it looks like it failed in my project for a different reason than in the template.

The only thing I changed is the way I add my UserStore in ConfigureServices.

Basically I was doing this:

`services.AddDefaultIdentity<CusomUser>().AddUserStore<CustomUserStore>().AddEntityFrameworkStores<CustomDbContext>();`

That didn't work. Then I changed it at some point to:

`services.AddDefaultIdentity<CusomUser>().AddUserStore<CustomUserStore>();`

Then it started working. I'm not even sure if I'm configuring the services correctly.

At a high level, what I'm trying to do is modify the provided EF Core UserStore::CreateAsync call to add some custom logic after the base method completes. Ie. override it and add stuff to it.

I did so by creating CustomUserStore that inherits from EF Core's UserStore then doing an override and configuring my services as above. Is that the recommended way to do it?

[Documentation ](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity-custom-storage-providers?view=aspnetcore-3.0)doesn't describe how to just add some small logic on top of all of the existing default behavior and default components.
